TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -86,6 +94,16 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
             self.initializeTicket()
         } else {
             self.reloadTicketRows()
+            if let streamingMarketDataService = streamingMarketDataService,
+                let symbol = self.order.symbol {
+                streamingMarketDataService.startUpdatingQuote(forSymbol: symbol, onUpdate: updateQuotePrice, onFailure: { self.clearMarketData() })
+            }
+        }
+    }
```

Comment:
Does this handle if the symbol is changed from the ticket?

Reply:
It does but the logic was: 
1. Start streaming service on view did appear
2. Start streaming service on Symbol change. 

So if the user changed the symbol and then went back to the view then 1 and 2 will be called. I added a check to prevent double `start` calls, I choose a simple approach by tracking `selectedSymbolChanged`.